types/video-react/index.d.ts

```diff
@@ -0,0 +1,510 @@
+// Type definitions for video-react 0.15.0
```

Comment:
The patch version shouldn't be included in the header here, only major and minor:

```suggestion
// Type definitions for video-react 0.15
```